--- templates/default/tmpl/details.tmpl	2013-09-10 09:25:37.331001371 +0200
+++ templates/default/tmpl/details.tmpl.new	2013-09-10 09:25:53.240003348 +0200
@@ -62,13 +62,6 @@
     <dd class="tag-default"><ul class="dummy"><li><?js= data.defaultvalue ?></li></ul></dd>
     <?js } ?>
     
-    <?js if (data.meta) {?>
-    <dt class="tag-source">Source:</dt>
-    <dd class="tag-source"><ul class="dummy"><li>
-        <?js= self.linkto(meta.filename) ?>, <?js= self.linkto(meta.filename, 'line ' + meta.lineno, null, 'line' + meta.lineno) ?>
-    </li></ul></dd>
-    <?js } ?>
-    
     <?js if (data.tutorials && tutorials.length) {?>
     <dt class="tag-tutorial">Tutorials:</dt>
     <dd class="tag-tutorial">
